webextension/scripts/utils.js

```diff
@@ -458,6 +459,14 @@ function notify(message, callback) {
   chrome.notifications && chrome.notifications.create(options, callback)
 }
 
+function checkLastError() {
+  if (chrome.runtime.lastError && chrome.runtime.lastError.message.startsWith("No tab with id:")) {
```

Comment:
I don't understand the intent of this line: why do we want only log the `lastError` if it's a tab error, but ignore it in case of other errors?

Reply:
I have updated the function in the latest commit. Please check this out. Thanks